src/chrome/content/rules/tbz.ch.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Invalid certificate: 
+		- webmail.tbz.ch
+    - vpn.tbz.ch
```

Comment:
Please only use tabs to indent lines and see also `intranet.tbz.ch`

Reply:
Thank you for the heads up. Fixed it now.